pkg/agent/controller/traceflow/traceflow_controller.go

```diff
@@ -293,6 +296,17 @@ func (c *Controller) validateTraceflow(tf *opsv1alpha1.Traceflow) error {
 	if tf.Spec.Destination.Service != "" && !features.DefaultFeatureGate.Enabled(features.AntreaProxy) {
 		return errors.New("using Service destination requires AntreaProxy feature enabled")
 	}
+	if tf.Spec.Destination.IP != "" {
+		destIP := net.ParseIP(tf.Spec.Destination.IP)
+		if destIP == nil {
+			return fmt.Errorf("destination IP is not valid: %s", tf.Spec.Destination.IP)
+		}
+		// When AntreaProxy is enabled, serviceCIDR is not required and may be nil. So we
```

Comment:
I think at least for not it cannot be nil, as we already set to the default value.

Reply:
That's right, I updated the comment :)